wiki/tiddlers/FAQ.tid

```diff
@@ -2,47 +2,46 @@ title: FAQ
 tags: odd
 
 !! I love this! Can I use for production/very important things?
-I would love to answer yes, but for now please don't do it. Noteself is still on Beta stage, and has to be extensively tested.
-But, if you are an early adopter and want to try new and exciting technologies NoteSelf is stable enough to give it a try.
+We would love to answer yes, but for now please don't do it. Noteself is still in Beta stage and has to be extensively tested.
+But if you are an early adopter and want to try new and exciting technologies NoteSelf is stable enough to give a try.
 
 !!Where is my data stored?
-Your data is stored inside your browser. We use PouchDB in the background,
-which creates a database on your browser's LocalStorage.
+Your data is stored inside your browser. We use PouchDB in the background, which creates a database in your browser's LocalStorage.
```

Comment:
I'm curious about why you have joined this into one line. Tiddlywiki treats line breaks as spaces, which was my intention